bigquery_etl/feature_usage/templates/metadata.yaml

```diff
@@ -14,7 +14,7 @@ labels:
   application: firefox
   schedule: daily
 scheduling:
-  dag_name: bqetl_main_summary
+  dag_name: bqetl_feature_usage
```

Comment:
I'm slightly confused by this, it looks like we're already scheduling this as of the part `bqetl_feature_usage` dag? Is this just updating the definition to be in sync?

Reply:
Yeah exactly, `bqetl_feature_usage` is the production dag. I think the prod dag was updated manually without the template being updated.